tox.ini

```diff
@@ -24,11 +24,35 @@ deps =
 commands =
     towncrier {posargs}
 
-[testenv:docs]
+[docs]
+change_dir = docs
 extras =
     docs
+# -v: verbose logging
+# -W: make warnings into errors
+# --keep-going: continue on errors
+# -E: force rebuild of environment
+# -a: read/parse all files
+# -n: nitpick mode
+sphinx_verbose_args = -v -W --keep-going -E -a -n
+
+[testenv:docs]
+change_dir = {[docs]change_dir}
+extras = {[docs]extras}
+commands =
+    sphinx-build -W -b html . _build/html
```

Comment:
While `sphinx-build` is installed as an entry point, I prefer the `python -m ` entry point because of the [issues associated with execution on Windows](https://snarky.ca/why-you-should-use-python-m-pip/)

Reply:
Also - would it be worth turn on `linkcheck` here? It doesn't seem overly expensive, and it would seem beneficial to verify that all links are valid as part of a "default" docs build.

Reply:
> While `sphinx-build` is installed as an entry point, I prefer the `python -m ` entry point because of the [issues associated with execution on Windows](https://snarky.ca/why-you-should-use-python-m-pip/)

I agree; although, `python -m sphinx-build` doesn't work. Based on my testing, `python -m sphinx` seems to default to the same behavior as a naked `sphinx-build` call....but I didn't dig in to confirm.

> Also - would it be worth turn on `linkcheck` here? It doesn't seem overly expensive, and it would seem beneficial to verify that all links are valid as part of a "default" docs build.

I guess it depends on how you want to think about these commands.

The `linkcheck` doesn't actually build the docs; it just checks the links. So, if we add the link checking here, it'll need to run before or after the existing call to actually build the docs. My thought was this might be annoying if you just want to quickly see your updates in HTML.

Reply:
Ah - that nitpicking technicality detail of "it doesn't work" :-)

As for a having a "fast build" option - that definitely makes sense. I was basing my suggestion on the fact that `linkcheck` was returning fairly quickly for me - but that appears to be because the `flathub.org` link was causing a fast fail. Once I fixed that, I can see a link check takes a moment (which shouldn't be surprising, given what it needs to do), and it doesn't seem to restrict the link check to files that have changed. 

On that basis, I agree that `docs` makes sense as a "fast path", with `docs-lint` being a full clean environment lint check.

On a related note, we probably want to add a note about these new options in the docs contribution guide, so that contributors know how to reproduce the failures they're likely to see in CI.

Reply:
Updated docs.